lib/lua/CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ mark_as_advanced(LUA_USE_DLOPEN)
 option(LUA_ANSI "Disable non-ANSI features." ${DEFAULT_ANSI})
 mark_as_advanced(LUA_ANSI)
 
+option(LUA_COMPILE_AS_C "Compile Lua as C rather than C++." FALSE)
+mark_as_advanced(LUA_COMPILE_AS_C)
```

Comment:
This should be a default, please remove the option.

Reply:
Should I replace it with an option called `LUA_COMPILE_AS_CXX`? Or should I simply remove the option?

Reply:
None at all.